src/index.js

```diff
@@ -202,7 +197,6 @@ export default {
           cwd,
           stdin,
           stream: 'both',
-          timeout: this.linterTimeout,
```

Comment:
I know, my point is that if you remove the timeout definition here then the default one of 10 seconds will be used and you will be getting timeout messages even with a unique key. Set the timeout to `Infinity` or something _very_ long but still finite (I've been using 5 minutes elsewhere).